src/cmap/commands.ts

```diff
@@ -705,7 +705,10 @@ export class OpMsgResponse {
     this.parsed = true;
   }
 
-  parseBsonSerializationOptions({ enableUtf8Validation }: BSONSerializeOptions): {
+  parseBsonSerializationOptions(
+    this: void,
```

Comment:
just curious - why is this necessary?

Reply:
It is not, I will revert. I did it when changing the utf8 tests over to spy on this method and noticed the method isn't instance-dependent.